### PR TITLE
Fix readme grammatical errors

### PR DESCRIPTION
**Pull request by anirudhtechandscience**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ In the first time you launch the software, it will automatically download models
 
 If you already have these files, you can copy them to the above locations to speed up installation.
 
-Below is a test on a relatively low-end laptop with **16GB System RAM** and **6GB VRAM** (Nvidia 3060 laptop). The speed on this machine is about 1.35 seconds per iteration. Pretty impressive – nowadays laptops with 3060 are usually at very acceptable price.
+Below is a test on a relatively low-end laptop with **16GB System RAM** and **6GB VRAM** (Nvidia 3060 laptop). The speed on this machine is about 1.35 seconds per iteration. Pretty impressive – nowadays laptops with a 3060 are usually at a acceptable price.
```
Comment (thinkyhead) [inline readme.md:41]: "an"

File: readme.md [hunk 2]
```diff
@@ -53,21 +53,21 @@ Note that the minimal requirement is **4GB Nvidia GPU memory (4GB VRAM)** and **
...
 
 (Last tested - 2023 Aug 14)
 
-| Colab | Info
+| Google Colab | Info
 | --- | --- |
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/lllyasviel/Fooocus/blob/main/colab.ipynb) | Fooocus Colab (Official Version)
 
-Note that sometimes this Colab will say like "you must restart the runtime in order to use newly installed XX". This can be safely ignored.
+Note that sometimes Colab will say like "you must restart the runtime in order to use newly installed XX". This can be safely ignored.
```
Comment (thinkyhead) [inline readme.md:64]: "will say *something* like"

File: readme.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ Coming soon ...
 ## List of "Hidden" Tricks
 <a name="tech_list"></a>
 
-Below things are already inside the software, and **users do not need to do anything about these**.
+Below things are already in the software, and **users do not need to do anything about these**.
```
Comment (thinkyhead) [inline readme.md:93]: Either "Things below" or "Items below" is better than "Below things."

File: readme.md [hunk 4]
```diff
@@ -106,7 +106,7 @@ Note that some of these tricks are currently (2023 Aug 11) impossible to reprodu
 
 ## Thanks
 
-The codebase starts from an odd mixture of [Automatic1111](https://github.com/AUTOMATIC1111/stable-diffusion-webui) and [ComfyUI](https://github.com/comfyanonymous/ComfyUI). (And they both use GPL license.)
+The codebase is based on a from an odd mixture of [Automatic1111](https://github.com/AUTOMATIC1111/stable-diffusion-webui) and [ComfyUI](https://github.com/comfyanonymous/ComfyUI). (And they both use the GPL license.)
```
Comment (thinkyhead) [inline readme.md:109]: "is based on a from an …" should be corrected to "is based on an …"

**Comment by mashb1t** [comment]
superseded by https://github.com/lllyasviel/Fooocus/pull/1655

**Pull request closed without merging**